soh/soh/z_message_OTR.cpp

```diff
@@ -1,18 +1,22 @@
 #include "OTRGlobals.h"
 #include "ResourceMgr.h"
 #include "Scene.h"
-#include "message_data_static.h"
 #include "Utils/StringHelper.h"
 #include "global.h"
 #include "vt.h"
 #include <Text.h>
+#include <message_data_static.h>
+#include <soh/Enhancements/custom_message/CustomMessageManager.h>
+#include <soh/Enhancements/custom_message/CustomMessageTypes.h>
 
 extern "C" MessageTableEntry* sNesMessageEntryTablePtr;
 extern "C" MessageTableEntry* sGerMessageEntryTablePtr;
 extern "C" MessageTableEntry* sFraMessageEntryTablePtr;
 extern "C" MessageTableEntry* sStaffMessageEntryTablePtr;
 //extern "C" MessageTableEntry* _message_0xFFFC_nes;	
 
+const std::string customMessageTableID = "BaseGameOverrides";
```

Comment:
this goes along with the last comment i made, maybe just moving this to `CustomMessageTypes.h` would be good?

Reply:
The idea is that when mods come around they will keep track of their own message table IDs in a similar way, so hardcoding that in the custom message system itself doesn't quite feel right.

Reply:
yeah, not sure where the best place for it would be. i think i've just dealt with a lot of string literals needing to match issues lately and i'm hoping to find a way to turn things into compile time errors instead of runtime errors

Reply:
Yeah the idea is that when these are initially defined is the only time you'll ever see a string literal. That one from one of your other comments was just something I forgot to replace with the variable I think. I might also be able to refactor it to dynamically give a UUID whenever you create a message table and store that in the class instance that generated it somehow, but that's still requiring strings to match at the end of the day.